raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -1070,6 +1072,7 @@ contract TokenNetwork is Utils {
     {
         bytes32 unlock_key1;
         bytes32 unlock_key2;
+        require(participant1 != participant2);
```

Comment:
I specifically did not add any checks for the `getChannelInfo` getter, to make it able to retrieve info for any combination of identifier and addresses.
It does not change the state.
It can be used to make sure we do not have channel data that is not supposed to be there (e.g. some checks fail in state changing functions).

But I am open to suggestions.

Reply:
I also prefer to have a `getChannelIdentifier` getter without checks, but at the moment this was the best solution I had to integrate those checks while not triggering the compiler error about the local variables limit...

Reply:
> But giving the two same addresses make no sense as a combination, right?

Correct, but just because we enforce this with checks in other state-changing functions. With this, we can do random checks (contract calls) to see if anything is out of place - if needed.

> I have no strong opinions on this and can remove it if you want me to and move it to the raiden client.

I would prefer this, unless there is a better reason to add them.

Reply:
No there isn't. Amending the commit.